app/uk/gov/hmrc/helptosavefrontend/config/FrontendWiring.scala

```diff
@@ -60,22 +78,17 @@ trait WSHttpExtension extends WSGet with WSPost {
 }
 
 @Singleton
-class WSHttp extends WSGet with WSPut with WSPost with WSDelete with AppName with RunMode with WSHttpExtension {
```

Comment:
where are the `AppName` and  `RunMode` compensated ?  or we don't need them ?

Reply:
looks like we never needed them - at least everything still compiles when they're not there